build/get_ext_tools.sh

```diff
@@ -0,0 +1,23 @@
+EXT_DIR=ext_tools
```

Comment:
this is not correct bash file missing `#!/usr/bin/env bash`

Reply:
Fixed

Reply:
I was suggesting better solution
> Run the program through /usr/bin/env, the user does not need to find the location of the program in the system (because the location of the command or program may be different in different systems), as long as the program is in your $PATH;
>
>Another advantage of running the program through /usr/bin/env is that it will find and run the default version according to your environment, providing flexibility.
>
>The downside is that in a multi-user system, someone else puts a bash in your $PATH, and an error may occur.
>
>In most cases, /usr/bin/env is the preferred choice, because it provides flexibility, especially if you want to run this script in a different version; and the way to specify the specific location#!/usr/bin/bash , In some cases it is safer because it limits the possibility of code injection.
>
>Maybe /usr/bin/bash does not exist on some systems, but /bin/bash must exist. So /bin/bash is the preferred choice for display designation.

Reply:
I've just done copy-paste form other build scripts.
You're right, on the other hand:
- it's a build script intended to be executed in a specific environment
- this PR is not intended to fix all nuances of our toolchain